requirements.in

```diff
@@ -50,6 +50,7 @@ pickleshare==0.7.5
 posthoganalytics==1.4.7
 protobuf==3.13.0
 psycopg2-binary==2.8.6
+pyjwt==2.1.0
```

Comment:
Why using `2.1.0` while `2.4.0` is available?

Reply:
I only hardcoded it to the existing version in requirements.txt (its a dependency of some other lib). I'll push it up